frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -260,6 +267,9 @@ export function AsyncMigrations(): JSX.Element {
                                 dataSource={asyncMigrations}
                                 expandable={rowExpansion}
                             />
+                            {activeAsyncMigrationModal ? (
```

Comment:
nit: naming - "active" is a weird term for a modal. IIRC we mostly use the term "show" in the codebase